### PR TITLE
`websit-scaffold`: auto generate possible value part.

### PR DESCRIPTION
**Pull request by wuxu92**
use the values in `ValidationFunc` to generate the document if the func is `StringInSlice`.

### CODE REVIEW

File: internal/tools/website-scaffold/main_test.go [hunk 1]
```diff
@@ -86,17 +87,19 @@ A 'block3' block supports the following:
...
 				Type:     schema.TypeList,
 				Required: true,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"nest_attr2": {
-							Type:     schema.TypeString,
-							Optional: true,
+							Type:         schema.TypeString,
+							Optional:     true,
+							ValidateFunc: validation.StringInSlice([]string{"foo", "bar"}, false),
```
Comment (tombuildsstuff) [inline internal/tools/website-scaffold/main_test.go:102]: since we're supporting it above, can we add a test above for the `"Foo", "Bar" and "Bar"` case?

**Review by tombuildsstuff — APPROVED** [review]
A couple of minor comments but this other LGTM - thanks for this @wuxu92

File: internal/tools/website-scaffold/main_test.go (hunk 1 above)
Comment (wuxu92) [reply internal/tools/website-scaffold/main_test.go]: i have updated this PR to add a new test-case, but i wonder if my understad is correct... so could you please have a look at it again 💭

**Review by tombuildsstuff — APPROVED** [review]
LGTM - thanks for pushing those changes @wuxu92

**Pull request merged**